fuzz.c

```diff
@@ -63,6 +63,22 @@ static void fuzz_getFileName(honggfuzz_t * hfuzz, char *fileName)
              (int)getpid(), (unsigned long int)tv.tv_sec, util_rnd64(), hfuzz->fileExtn);
 }
 
+void fuzz_getExtension(const char *file_name,char *extension)  
+{  
+    int i=0,length;  
+    length=strlen(file_name);  
+        while(file_name[i])  
```

Comment:
You don't need to loop around the entire string. An `strrchr` call against the dot char `.` will be enough to have the minimum perf overhead.